app/web/markdown/blog.md

```diff
@@ -6,6 +6,14 @@ title: Blog
 
 If you'd like to contribute to the blog, please [sign up](/volunteer) and let us know!
 
+## [Hosting refugees and finding shelter with Couchers.org](/blog/2022/03/10/hosting-refugees-and-finding-shelter-with-couchers-org-en)
```

Comment:
missing other languages here

also dates are off now

Reply:
I don't think we should list all languages on the blog main page. I think links on the individual blogs is sufficient.

Reply:
why? if i speak english i'll just scroll past